README.md

```diff
@@ -25,7 +25,8 @@ If you have questions or need help, please check out our documentation for a [li
     Run the following commands.
     - `conda env create -f ./.ci_scripts/environment.yml`
     - `conda activate conda-forge-docs`
-    - For live builds, `npm install && npm run start`
+    - enable pre-commit hooks with `pre-commit install`
```

Comment:
```suggestion
    - Optional. Enable pre-commit hooks with `pre-commit install`.
```

Out of personal preference 😬 It's fine to also fix in CI via the `pre-commit.ci` commands.